application/classes/Ushahidi/Validator/Form/Update.php

```diff
@@ -31,12 +31,27 @@ public function __construct(FormRepository $repo)
 
 	protected function getRules()
 	{
+
+			// Always check validation for name and description
+			$fullData = $this->validation_engine->getFullData();
+			$data = $this->validation_engine->getData();
+			if ($fullData['name']) {
+				$data['name'] = $fullData['name'];
+			}
+			if ($fullData['description']) {
+				$data['description'] = $fullData['description'];
+			}
+			$this->validation_engine->setData($data);
+			// End
+
 		return [
 			'name' => [
+				['not_empty'],
 				['min_length', [':value', 2]],
```

Comment:
If the field is left empty, `min_length` does not throw the error, allowing for the name to be saved as an empty string, which is why I moved `not_empty` to `Form/Update.php`.